CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog of postpic
 
+## v0.3
+2017-09-28
+
+Many improvements in terms of speed and features. Unfortunately some changes are not compatible to v0.2.3.
+
+**Highlights**
+* kspace reconstruction and propagation of EM waves.
+* `postpic.Field` properly handles operator overloading and slicing. Slicing can be index based (integers) or referring the actual physical extent on the axis of a Field object (using floats).
+* Expression based interface to particle properties (see below)
+
+**Incompatible adjustments to last version**
+* New dependency: Postpic requires the `numexpr` package to be installed now.
```

Comment:
Then maybe we should move the `import numexpr` to the top of all modules which use it instead of still having it inside the functions